components/Grid/grid.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Grid</title>
+    <link rel="stylesheet" href="grid.css">
+</head>
+<body>
+    <div class="high-container">
+
+
+   
+    <h3>50:50 Grid example</h3>
+    <div class="grid-50-50">
+        <img src="/images/scorpio-sign.jpg" alt="">
```

Comment:
add accessibility text in your alt attribute

Reply:
sure ,will do that now .